website/docs/stitch-type-merging.md

```diff
@@ -8,8 +8,6 @@ Type merging allows _partial definitions_ of a type to exist in any subschema, a
 
 Type merging is now the preferred method of including GraphQL types across subschemas, replacing the need for [schema extensions](/docs/stitch-schema-extensions) (though does not preclude their use). To migrate from schema extensions, simply enable type merging and then start replacing extensions one by one with merges.
 
-<div class="video-player"><iframe src="https://www.youtube.com/embed/KBACiSA5sEQ?list=PLTJ2vmU3jbWy6JntdRQZAmy0mcpYu2OD1" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
```

Comment:
Coming at this video placement with fresh eyes... it's a bit too specific to one chapter topic to holistically describe type merging, so probably isn't doing any favors here as an intro video. Just navigating from docs > handbook > chapter video seems like the most assured path for delivering relevant information.